EventVisualisation/Base/src/DataInterpreter.cxx

```diff
@@ -28,7 +27,7 @@ DataInterpreter::DataInterpreter() = default;
 
 TEveElement* DataInterpreter::interpretDataForType(TObject* /*data*/, EVisualisationDataType /*type*/)
 {
-  cout << "Virtual method interpretDataForType(EventManager::EDataType type) -- should be implemented in deriving class!!" << endl;
+  LOG(FATAL) << "Virtual method interpretDataForType(EventManager::EDataType type) -- should be implemented in deriving class!!";
```

Comment:
why not make this virtual method pure abstract? Then there is a compile time guarantee that this function is implemented in a deriving class.